packages/select-react/package.json

```diff
@@ -1,13 +1,14 @@
 {
-    "name": "@fremtind/jkl-dropdown-react",
-    "version": "0.9.0",
+    "name": "@fremtind/jkl-select-react",
+    "version": "0.0.0",
     "publishConfig": {
         "access": "public"
     },
-    "description": "Jøkul react dropdown component",
+    "description": "Jøkul react select component",
     "homepage": "https://fremtind.github.io/jokul",
     "keywords": [
         "dropdown",
```

Comment:
Don't know if this affects anything but remove dropdown keyword?

Reply:
Replace all is _very_ effective 😅 Should probably add back "dropdown" as a keyword, yes.

Reply:
I think this is just for searching on npmjs and so on. I'm thinking that it should stay, maybe? I don't think we're the only ones mixing up these two components 😅